### PR TITLE
Move to Cloudflare Pages

### PR DESCRIPTION
**Pull request by mhart**
A couple of things to note:

1. This keeps `wrangler.toml` so that local dev works... not sure if this is a bug or feature (will check with the Pages team EDIT: turns out it's a bug, ish – at least, isn't really intended to work and shouldn't be relied on, but is ok for local dev)
2. Got rid of `--watch` cause it was a bit messy with building css and js, but could add that in with a separate `build:watch` or something

### CODE REVIEW

**Comment by mhart** [comment]
Oh yeah, need to add instructions to add the `AI` binding for when you deploy. As per [the official Pages documentation](https://developers.cloudflare.com/pages/functions/bindings/#workers-ai):

Screenshot 2024-02-22 at 6 04 15 pm

File: src/index.js [hunk 1]
```diff
@@ -31,4 +27,6 @@ app.post('/api/chat', async (c) => {
 	});
 });
 
+app.get('/*', (c) => c.env.ASSETS.fetch(c.req.raw));
```
Comment (mhart) [inline src/index.js:30]: For reference: https://developers.cloudflare.com/pages/functions/advanced-mode/#set-up-a-function

File: assets/_routes.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+	"version": 1,
+	"include": ["/api/*"],
```
Comment (mhart) [inline assets/_routes.json:3]: This file isn't technically necessary, but does allow you to specify which routes are "free"/"static" and which ones will cost ya. See: https://developers.cloudflare.com/pages/functions/routing/#functions-invocation-routes

**Comment by craigsdennis** [comment]
Thanks @mhart !

**Pull request merged**